website/server/models/userNotification.js

```diff
@@ -5,73 +5,52 @@ import validator from 'validator';
 import baseModel from '../libs/baseModel';
 
 const NOTIFICATION_TYPES = [
-  'DROPS_ENABLED', // unused
-  'REBIRTH_ENABLED',
-  'WON_CHALLENGE',
-  'STREAK_ACHIEVEMENT',
-  'ULTIMATE_GEAR_ACHIEVEMENT',
-  'REBIRTH_ACHIEVEMENT',
-  'NEW_CONTRIBUTOR_LEVEL',
+  // general notifications
+  'BOSS_DAMAGE', // Not used currently but kept to avoid validation errors
+  'CARD_RECEIVED',
   'CRON',
+  'DROP_CAP_REACHED',
+  'DROPS_ENABLED', // unused
+  'FIRST_DROPS',
+  'GIFT_ONE_GET_ONE',
+  'GROUP_INVITE_ACCEPTED',
   'GROUP_TASK_APPROVAL',
   'GROUP_TASK_APPROVED',
   'GROUP_TASK_ASSIGNED',
   'GROUP_TASK_CLAIMED',
   'GROUP_TASK_NEEDS_WORK',
-  'LOGIN_INCENTIVE',
-  'GROUP_INVITE_ACCEPTED',
-  'SCORED_TASK',
-  'BOSS_DAMAGE', // Not used currently but kept to avoid validation errors
-  'GIFT_ONE_GET_ONE',
   'GUILD_PROMPT',
-  'GUILD_JOINED_ACHIEVEMENT',
-  'CHALLENGE_JOINED_ACHIEVEMENT',
-  'INVITED_FRIEND_ACHIEVEMENT',
-  'CARD_RECEIVED',
-  'NEW_MYSTERY_ITEMS',
-  'UNALLOCATED_STATS_POINTS',
+  'LEVELED_UP', // not in use
+  'LOGIN_INCENTIVE',
+  'NEW_CHAT_MESSAGE',
+  'NEW_CONTRIBUTOR_LEVEL',
   'NEW_INBOX_MESSAGE',
+  'NEW_MYSTERY_ITEMS',
   'NEW_STUFF',
-  'NEW_CHAT_MESSAGE',
-  'LEVELED_UP', // Not in use
-  'FIRST_DROPS',
   'ONBOARDING_COMPLETE',
-  'ACHIEVEMENT_ALL_YOUR_BASE',
-  'ACHIEVEMENT_BACK_TO_BASICS',
-  'ACHIEVEMENT_JUST_ADD_WATER',
-  'ACHIEVEMENT_LOST_MASTERCLASSER',
-  'ACHIEVEMENT_MIND_OVER_MATTER',
-  'ACHIEVEMENT_DUST_DEVIL',
-  'ACHIEVEMENT_ARID_AUTHORITY',
-  'ACHIEVEMENT_PARTY_UP',
-  'ACHIEVEMENT_PARTY_ON',
-  'ACHIEVEMENT_BEAST_MASTER',
-  'ACHIEVEMENT_MOUNT_MASTER',
-  'ACHIEVEMENT_TRIAD_BINGO',
-  'ACHIEVEMENT_MONSTER_MAGUS',
-  'ACHIEVEMENT_UNDEAD_UNDERTAKER',
-  'ACHIEVEMENT_PRIMED_FOR_PAINTING',
-  'ACHIEVEMENT_PEARLY_PRO',
-  'ACHIEVEMENT_TICKLED_PINK',
-  'ACHIEVEMENT_ROSY_OUTLOOK',
-  'ACHIEVEMENT_BUG_BONANZA',
-  'ACHIEVEMENT_BARE_NECESSITIES',
-  'ACHIEVEMENT_FRESHWATER_FRIENDS',
-  'ACHIEVEMENT_GOOD_AS_GOLD',
-  'ACHIEVEMENT_ALL_THAT_GLITTERS',
-  'ACHIEVEMENT_BONE_COLLECTOR',
-  'ACHIEVEMENT_SKELETON_CREW',
-  'ACHIEVEMENT_SEEING_RED',
-  'ACHIEVEMENT_RED_LETTER_DAY',
-  'ACHIEVEMENT_LEGENDARY_BESTIARY',
-  'ACHIEVEMENT_SEASONAL_SPECIALIST',
-  'ACHIEVEMENT_VIOLETS_ARE_BLUE',
-  'ACHIEVEMENT_WILD_BLUE_YONDER',
-  'ACHIEVEMENT_DOMESTICATED',
-  'ACHIEVEMENT_SHADY_CUSTOMER',
-  'ACHIEVEMENT_SHADE_OF_IT_ALL',
```

Comment:
ok, this is going to be a problem for anyone who happens to have some legacy data. It might be these notifications are too ephemeral to stick around, but have you queried the db to see if any of these are still out there? If they are, we can mark these as deprecated, but deleting them entirely will lead to `enum` errors.

Reply:
added back in with ` // deprecated`